versions.props

```diff
@@ -49,9 +49,9 @@ net.jpountz.lz4:lz4 = 1.3.0
 one.util:streamex = 0.7.3
 org.apache.commons:commons-lang3 = 3.12.0
 org.apache.commons:commons-pool2 = 2.10.0
-org.apache.thrift:libthrift = 0.12.0
+org.apache.thrift:libthrift = 0.14.2
```

Comment:
Thrift 0.14 moved some classes around, especially in https://github.com/apache/thrift/pull/2191 notably moving `org.apache.thrift.transport.TFramedTransport` to `org.apache.thrift.transport.layered.TFramedTransport` which [causes symbol error in build](https://app.circleci.com/pipelines/github/palantir/atlasdb/4450/workflows/f8291964-6403-43ee-bd40-332f91a5e7d9/jobs/23851)

[Thrift changelog](https://github.com/apache/thrift/blob/master/CHANGES.md)

```
> Task :atlasdb-cassandra:compileJava
/home/circleci/project/atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraClientPoolingContainer.java:51: error: cannot find symbol
import org.apache.thrift.transport.TFramedTransport;
                                  ^
  symbol:   class TFramedTransport
  location: package org.apache.thrift.transport
/home/circleci/project/atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraClientFactory.java:54: error: cannot find symbol
import org.apache.thrift.transport.TFramedTransport;
                                  ^
  symbol:   class TFramedTransport
  location: package org.apache.thrift.transport
2 errors

> Task :atlasdb-cassandra:compileJava FAILED
```

Reply:
@tpetracca for SA as follow up to https://github.com/palantir/atlasdb/pull/5522

Reply:
pushed [`c196f90` (#5537)](https://github.com/palantir/atlasdb/pull/5537/commits/c196f9011dde39a4394f40051231d724df70fad4)